README.md

```diff
@@ -313,9 +323,6 @@ Using `withOptions` will **not** modify the options initially provided, but retu
   const { events } = init({ apiKey: '<YOUR_API_KEY>' });
 
   const options: FSRequestOptions = { 
-    // integrationSource is generally intended for FullStory
-    // or FullStory partners while building integrations.
-    //Generally it should be left empty.
```

Comment:
This is a good doc string to include in the source code - so that it shows up in IDEs